### PR TITLE
[spaceship] add ability to delete multiple beta testers at a time

### PR DESCRIPTION
**Pull request by csc-EricWu**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

**Comment by csc-EricWu** [comment]
Please provide exact steps to test this. So far I can only see how to delete testers. Also we cannot add PRs that are not matching our code style.

@max-ott I changed to the same style as you. I provide an API for deleting multiple testers at one time. You should know that you need an account to have a case.

**Comment by csc-EricWu** [comment]
@joshdholtz  Can you have a look? I need this API please

**Comment by joshdholtz** [comment]
I have some family things I need to do this weekend but I will look on Monday!

**Comment by csc-EricWu** [comment]
ok, Thank you for your reply.

File: spaceship/lib/spaceship/connect_api/testflight/testflight.rb [hunk 1]
```diff
@@ -277,6 +277,19 @@ def delete_beta_tester_from_beta_groups(beta_tester_id: nil, beta_group_ids: [])
           test_flight_request_client.delete("betaTesters/#{beta_tester_id}/relationships/betaGroups", nil, body)
         end
 
+        def delete_beta_testers_from_app(beta_tester_ids: [], app_id: nil)
```
Comment (joshdholtz) [inline spaceship/lib/spaceship/connect_api/testflight/testflight.rb:280]: The title of the PR is "add and delete" but this is named `delete_beta_testers_from_app` 🤔 Should this method actually be called something like `update_beta_testers_from_app`?

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Thanks for making this PR! Just one change request on the name of the new method (I think) 😇 Let me know your thoughts!

File: spaceship/lib/spaceship/connect_api/testflight/testflight.rb (hunk 1 above)
Comment (csc-EricWu) [reply spaceship/lib/spaceship/connect_api/testflight/testflight.rb:280]: It's just that you can delete more than one tester at a time, which is very useful to me. My English is not very good. I'm sorry.

Comment (csc-EricWu) [reply spaceship/lib/spaceship/connect_api/testflight/testflight.rb:280]: Delete multiple testers, which greatly improves efficiency.

**Comment by joshdholtz** [comment]
@googlebot I consent.

**Review by joshdholtz — APPROVED** [review]
Thanks so much for adding this! This should get released later this week. I really appreciate the contribution ❤️

**Pull request merged**